src/main/java/com/example/taskmanagementapplication/model/Attachment.java

```diff
@@ -0,0 +1,45 @@
+package com.example.taskmanagementapplication.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Table(name = "attachments")
+@SQLDelete(sql = "UPDATE book SET is_deleted = 1 WHERE id = ?")
+@SQLRestriction("is_deleted = false")
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+public class Attachment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @EqualsAndHashCode.Exclude
+    @ToString.Exclude
+    @JoinColumn(name = "task_id", nullable = false)
+    private Task task;
+    @Column(nullable = false)
+    private String dropboxFileId;
+    @Column(nullable = false)
+    private String fileName;
+    @Column(nullable = false)
+    private LocalDateTime uploadDate;
+    @Column(nullable = false)
+    private boolean isDeleted;

```

Comment:
use @SoftDelete annotation to get rid of the excess field in the model
![image](https://github.com/jv-feb24-team04/task-management-app/assets/159646520/7bb89c6c-3ddd-4457-acce-32ace8bf551d)